SpatialGDK/Source/SpatialGDK/Private/Interop/Startup/ServerWorkerStartupCommon.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "DefaultServerWorkerStartupHandler.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Interop/SkeletonEntityCreationStep.h"
+#include "Interop/Startup/SpatialStartupCommon.h"
+
+namespace SpatialGDK
+{
+struct FInternalState
```

Comment:
Inherits from TSharedFromThis.
And maybe a less generic name is in order? FStartupInternalState? FStrateupSharedContext?